.github/workflows/workflow.yml

```
@@ -225,33 +214,31 @@ jobs:
   test-on-windows:
     runs-on: "windows-latest"
     if: github.ref == 'refs/heads/master' # Only triggered for the `master` branch
-    needs: [ build-conda ]
+    needs: [ test-conda ]
     steps:
+      # Checkout needed to get github.sha
+      - uses: actions/checkout@v4
       - uses: conda-incubator/setup-miniconda@v3
         with:
-          auto-update-conda: true
-          python-version: "3.9.9"
-          # Add conda-forge for OpenFisca-Core
-          channels: openfisca,conda-forge
-          activate-environment: true
-      - uses: actions/checkout@v4
+          activate-environment: openfisca
+          miniforge-version: latest
+          python-version: 3.9.12
+          use-mamba: true
+      - name: Restore build
+        uses: actions/cache@v4
+        with:
+          path: /tmp/conda-bld
+          key: build-conda-${{ github.sha }}
+          fail-on-cache-miss: true
+          enableCrossOsArchive: true  # To allow restore on windows
       - name: Test max path length
         run: "python3 openfisca_france/scripts/check_path_length.py"
-      - name: Get version
-        run: |
-          # chcp 65001 #set code page to utf-8
-          echo ("PACKAGE_VERSION=" + (python3 ./setup.py --version) ) >> $env:GITHUB_ENV
-          echo "Version setup.py: ${{ env.PACKAGE_VERSION }}"
-      - name: Download conda build
-        uses: actions/download-artifact@v3
-        with:
-          name: conda-build-${{ env.PACKAGE_VERSION }}-${{ github.sha }}
-          path: conda-build-tmp
-      - name: Install with conda
-        run: |
-          conda install -c ./conda-build-tmp/noarch/openfisca-france-dev-${{ env.PACKAGE_VERSION }}-py_0.tar.bz2  openfisca-france-dev
-      - name: openfisca test
-        run: openfisca test --country-package openfisca_france tests
+      - name: Install package
+        shell: bash -l {0}
```

Décrire l'intention sur la première occurrence de la commande dans le fichier a minima ?
```suggestion
        shell: bash -l {0}  # démarre un login shell avec les variables d'environnement
```

---

Fait